src/components/ItaliaTheme/View/Commons/Gallery.jsx

```diff
@@ -82,16 +88,17 @@ const Gallery = ({ content, folder_name }) => {
   }, [dispatch, content, url, folder_name]);
 
   const multimedia = searchResults?.[folder_name]?.items || [];
-  let images = multimedia.filter(item => item['@type'] === 'Image');
-  let videos = multimedia.filter(item => item['@type'] === 'Link');
+  let images = multimedia.filter((item) => item['@type'] === 'Image');
+  let videos = multimedia.filter((item) => item['@type'] === 'Link');
+
   return (
     <>
```

Comment:
aggiorna il manuale di tutti i ct in cui appare questo comp[onente per indicar che i link sono videp